dash/testing/browser.py

```diff
@@ -18,13 +19,13 @@
 
 from dash.testing.wait import text_to_equal, style_to_equal
 from dash.testing.dash_page import DashPageMixin
-from dash.testing.errors import DashAppLoadingError
+from dash.testing.errors import DashAppLoadingError, BrowserError
 
 
 logger = logging.getLogger(__name__)
 
 
-class Browser(DashPageMixin):
+class Browser(DashPageMixin):  # pylint: disable=too-many-instance-attributes
```

Comment:
A limit of 7 instance attrs seems a bit draconian. The only other place we disable this is on the `Dash` class itself, which used to have more but since the config refactor it has 16. Could just bump the limit up in pylintrc to accommodate both of these.